tests/src/io.jsonwebtoken/jjwt-gson/0.11.5/build.gradle

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+
+import org.graalvm.internal.tck.TestUtils
+
+plugins {
+    id "org.graalvm.internal.tck"
+}
+
+String libraryVersion = TestUtils.testedLibraryVersion
+
+dependencies {
+    testImplementation "io.jsonwebtoken:jjwt-gson:$libraryVersion"
+    testImplementation 'org.assertj:assertj-core:3.22.0'
+    testImplementation "io.jsonwebtoken:jjwt-api:$libraryVersion"
+    testImplementation "io.jsonwebtoken:jjwt-impl:$libraryVersion"
+}
```

Comment:
- Do I need to bring the `user-code-filter.json` that I personally use to collect GraalVM accessibility metadata into this PR?

Reply:
Yes, please do.

Reply:
- I have included this part.